### ISSUE TITLE
Allow admins to "pause" a job instead of just disabling it through the UI

### ISSUE THREAD

**Issue by will-lauer** [screwdriver-cd/screwdriver#2214]
Currently, in the Options tab in the UI, an user with appropriate privileges can mark specific jobs as disabled. This is handy if you need to run the pipeline up to a specific point but not to completion. For example, in a pipeline running deployments, this can be used to deploy to the backup site and stop, allowing some manual testing, prior to starting the next step to deploy to the primary site. The problem with the existing "disable" functionality is that when a job is re-enabled, pending invocations are not triggered.

What I'd like to see is a "pause" function in the UI, similar to the disable function, so that when a job is unpaused, any pending triggers are automatically fired. I would expect this to work something like the existing `freezeWindows` functionality: pausing a job would effectively start a freezeWindow and unpausing it would end the freezeWindow.

**Comment by jithine** [comment]
Does a concept of human trigger address this concerb https://github.com/screwdriver-cd/screwdriver/issues/1323#issuecomment-429667973 ?

**Comment by will-lauer** [comment]
I don't think so. #1323 asks for a permanent gate in the workflow - don't pass "go" until the user gives permission. I'm looking for a temporary block - usually my pipeline should build -> test -> deploy stage -> deploy backup -> deploy production. I don't want human intervention most of the time. But I do want to be able to say "oh, something risky is about to happen, so temporarily pause 'deploy production'" until I'm ready. And when I'm ready, I want deploy production to run automaticallys. Sure, I could manually click "run pipeline from here", but I have to remember to do that. And what if my pipeline doesn't consolidate multiple runs, and there are multiple triggers "pending". I would have go find all the triggering piplines and select "start pipeline from here" and hope they triggered in the correct order.

I do like the idea of #1323, and can see how that would be useful for a number of scenarios, but its doesn't solve the problem I'm trying to solve. 

Thinking about it a little more, I keep coming back back to the notion of freezeWindows. If I could manually create freezeWindows through the UI, and then update them, that would solve a number of problems. For jobs that I need to pause, I could create an open-ended freezeWindow (freeze forever), and then when I want to unpause them, I could either delete the freeze window or just update the end time. But this would also allow me to create other temporary freeze windows (its the start of earnings season, so don't deploy my financial app during the day, OR we are releasing a new product next week, so pause the regular deployment cycle of other products temporarily) without having to make commits to the source code control system.
